examples/hosted-delivery-function-templates/typescript/example.ts

```diff
@@ -22,18 +20,18 @@ const fieldMappingHandler: EventHandler<EventType.GRAPHQL_FIELD_MAPPING> = (even
   };
 };
 
-const queryHandler: EventHandler<EventType.GRAPHQL_QUERY> = (event, context) => {
+const queryHandler: EventHandler<'graphql.query'> = (event, context) => {
   return {
     data: {},
     errors: [],
   };
 };
 
 export const handler: EventHandler = (event, context) => {
-  if (event.type === EventType.GRAPHQL_FIELD_MAPPING) {
+  if (event.type === GRAPHQL_FIELD_MAPPING_EVENT) {
```

Comment:
You could replace this (and other occasions) with `graphql.field.mapping` as well.

Reply:
We have it as a `const` at the top - why against that?

Reply:
I don't see the point of introducing the constant in the first place. The event types are under our control and properly exposed via node-apps-toolkit, so there is no additional type safety to gain here. But feel free to move on.